src/utilities/savedFilters.ts

```diff
@@ -71,7 +71,9 @@ export const excludeScheduledSavedSearch = new SavedSearch({
   filters: noScheduleFilter,
 })
 
-export const defaultSavesSearches = [
+export const systemDefaultSavedSearch = oneWeekSavedSearch
+
+export const builtinSavedSearches = [
```

Comment:
nit: I like the `system` keyword and might carry that into this array and call them `systemSavedSearches`.